Source/SVONavigation/Private/SVOBoundsNavigationData.cpp

```diff
@@ -521,6 +525,7 @@ void FSVOBoundsNavigationData::BuildNeighborLinks( const LayerIndex layer_index
     QUICK_SCOPE_CYCLE_COUNTER( STAT_SVOBoundsNavigationData_BuildNeighborLinks );
 
     auto & layer_nodes = SVOData.GetLayer( layer_index ).GetNodes();
+    const auto max_layer_index = GetLayerCount() - 2;

```

Comment:
didn't notice this before, but why -2 actually?

Reply:
-1 to not go out of bounds and -1 because layer 0 does not count as it contains leaves and subnodes